browser/src/map/handler/Map.Keyboard.js

```diff
@@ -837,14 +847,7 @@ L.Map.Keyboard = L.Handler.extend({
 			this._map.print();
 			return true;
 		case this.keyCodes.S: // s
-			// Save only when not read-only and when HideSaveOption is false.
-			if (!this._map.isReadOnlyMode() && !this._map['wopi'].HideSaveOption) {
-				this._map.fire('postMessage', {msgId: 'UI_Save', args: { source: 'keyboard' }});
-				if (!this._map._disableDefaultAction['UI_Save']) {
-					this._map.save(false /* An explicit save should terminate cell edit */,
-					               false /* An explicit save should save it again */);
-				}
-			}
+			// Handle save event in globalEventHandler
```

Comment:
if we don't handle anything here, shouldn't we just skip this?

Reply:
true probably means - I handled that

Reply:
https://github.com/CollaboraOnline/online/pull/9999/files/1ccfe139e3d1c29ee565fc6cf72a105216920178#diff-5c4df738e4e71c87f575a954037607d6ffac23b63686d7de65dd029bca95eed6R533

@eszkadev it's a complicated implementaion of this._handleCtrlCommand. if we just skip it then it tries to exucute  rest  and we encounter erros like "User has no permission to save" or frquently we get "Content overwrite by other user overtite the current changes..." so it's a better way if we just return true and let _globalKeyEventHandler to handle "save"